backend/routes/admin.js

```diff
@@ -30,7 +30,7 @@ router.get("/candidate-submissions", function(req, res) {
     },
     {
       $project:{
-              candidateName: '$Organization Name (legal name)',
```

Comment:
@GodGreg Did the database column name change? I noticed the organization name wasn't showing up and changing this got the data to show up on the API calls.